src/sys/signal.rs

```diff
@@ -413,6 +413,8 @@ pub const SIGUNUSED : Signal = SIGSYS;
 cfg_if! {
     if #[cfg(target_os = "redox")] {
         type SaFlags_t = libc::c_ulong;
+    } else if #[cfg(all(target_env = "uclibc", target_arch = "mips"))] {
```

Comment:
```suggestion
    } else if #[cfg(all(target_env = "uclibc", any(target_arch = "mips"), target_arch = "mips64"))] {
```

In your libc PR, you did it to both `mips` and `mips64`